docs/conf.py

```diff
@@ -206,17 +124,20 @@
 html_context["current_language"] = current_language
 
 # SET CURRENT_VERSION
-from git import Repo
-
-repo = Repo(search_parent_directories=True)
-
 if "current_version" in os.environ:
```

Comment:
Shouldn't it be: if "GITHUB_REF_NAME" in os.environ:

Reply:
You are right. Good catch!